app/ai/voice/agents/breeze_buddy/template/loader.py

```
@@ -142,12 +145,26 @@ async def load_template(
                 logger.warning(f"Field '{field_name}' from schema not found in payload")
-                template_vars[field_name] = ""
+                value = ""
+            # Check for function in schema
+            function_name = None
+            if isinstance(field_schema, dict):
+                function_name = field_schema.get("function")
+            if function_name and function_name in TEMPLATE_FUNCTION_REGISTRY:
+                try:
+                    func = TEMPLATE_FUNCTION_REGISTRY[function_name]
+                    value = func(round(float(value)))
+                except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)